lib/omniauth/strategies/amber_oauth2.rb

```diff
@@ -21,10 +21,10 @@ def authorize_params
       info do
         {
           uid: raw_info['id'],
-          username: raw_info['attributes']['username'],
           name: full_name(raw_info),
-          avatar_url: raw_info['attributes']['avatar_url'],
+          email: raw_info['attributes']['email'],
```

Comment:
This field is used but was previously not added to the user during user creation or updated during login